lib/storage/client.js

```diff
@@ -481,6 +481,43 @@ export default class Client extends CryptoConsumer {
     return this.internalReadNoteByName(anonAuth, noteName, encryptionKeyPair, apiUrl)
   }
 
+  async replaceNoteByName(data, recipientEncryptionKey, recipientSigningKey, options) {
+    // Automatically mix in the client id unless it is overridden in the provided options.
```

Comment:
Yes but why? Remember good comments extend beyond the how of the code and into the why of the code (useful for the next developer to know when and how to safely make changes to the code without introducing subtle bugs because they've violated some implicit *cough* undocumented *cough* assumption the code makes)

Reply:
noted and improved (the comment was a copy pasta from the writeNote method)